src/app-init/apiManager.js

```diff
@@ -7,8 +7,13 @@ import { ROUTE_HOME, ROUTE_DASHBOARD } from 'app-init/router';
 import pluginsArray from 'entando-plugins';
 
 config(store, ROUTE_HOME, ROUTE_DASHBOARD);
+let parsedDomain = process.env.DOMAIN;
```

Comment:
1) Why calling `parsedDomain` a variable that's not processed in any way?
2) Why calling `embeddedDomain` a variable that represents an URL path, and not a domain?
3) Why not using a ternary like this?
```
const domain = process.env.EMBEDDED_DOMAIN ?
  `${window.location.origin}/${process.env.EMBEDDED_DOMAIN}` :
  process.env.DOMAIN;
```